### ISSUE TITLE
The Buidler EVM tracing engine could not be initialized

### ISSUE THREAD

**Issue by ZeframLou** [nomiclabs/buidler#593]
I am getting the error below when I run `npx buidler test` in my project:
```
The Buidler EVM tracing engine could not be initialized. Run Buidler with --verbose to learn more.
  buidler:core:buidler-evm:node Buidler EVM tracing disabled: ContractsIdentifier failed to be initialized. Please report this to help us improve Buidler.
  buidler:core:buidler-evm:node  TypeError: Cannot read property 'location' of undefined
    at uncompressSourcemaps (/home/zefram/projects/88mph-contracts/node_modules/@nomiclabs/buidler/src/internal/buidler-evm/stack-traces/source-maps.ts:39:31)
    at Object.decodeInstructions (/home/zefram/projects/88mph-contracts/node_modules/@nomiclabs/buidler/src/internal/buidler-evm/stack-traces/source-maps.ts:64:22)
    at decodeEvmBytecode (/home/zefram/projects/88mph-contracts/node_modules/@nomiclabs/buidler/src/internal/buidler-evm/stack-traces/compiler-to-model.ts:341:24)
    at decodeBytecodes (/home/zefram/projects/88mph-contracts/node_modules/@nomiclabs/buidler/src/internal/buidler-evm/stack-traces/compiler-to-model.ts:304:32)
    at Object.createModelsAndDecodeBytecodes (/home/zefram/projects/88mph-contracts/node_modules/@nomiclabs/buidler/src/internal/buidler-evm/stack-traces/compiler-to-model.ts:39:21)
    at new BuidlerNode (/home/zefram/projects/88mph-contracts/node_modules/@nomiclabs/buidler/src/internal/buidler-evm/provider/node.ts:314:25)
    at Function.create (/home/zefram/projects/88mph-contracts/node_modules/@nomiclabs/buidler/src/internal/buidler-evm/provider/node.ts:224:18) +0ms
```

### Package versions
```
"@nomiclabs/buidler": "^1.3.4",
"@nomiclabs/buidler-truffle5": "^1.3.3",
"@nomiclabs/buidler-web3": "^1.3.3"
```

### Reproduction steps
1. Clone [this repo](https://github.com/Bacon-labs/88mph-contracts/tree/longing)
2. Switch to branch `longing`
3. Do `npm install`
4. Run `npx buidler test --verbose`

**Comment by fvictorio** [comment]
Hey @ZeframLou, I couldn't reproduce this with those steps. I get the warning, but not the errors. Is it possible that you have fixed it since you opened the issue? Or it's still happening and I'm not being able to reproduce it?

**Comment by fvictorio** [comment]
Oh wait, I forgot the `--verbose`. I get it now.

**Comment by fvictorio** [comment]
To whomever works on this: the problem seems to be that one of the sourcemaps (the one for ERC20Mock) starts like this:

```
:::-;146:146:23;416:163:7;;;;...
```

I thought this wasn't supposed to be valid, because empty fields are interpreted with the same value as the one in the previous element, so the first one shouldn't be empty. The code seems to make the same assumption:

```
          parts[0] !== undefined && parts[0] !== ""
            ? +parts[0]
            : mappings[i - 1].location.offset,
```

since that can be executed when `i === 0`, which is what it seems to be happening here.

I guess a simple solution is to add a `i === 0` condition there, but what value is supposed to take?

**Comment by alcuadrado** [comment]
This was fixed in #655.

Thanks @ZeframLou for the detailed reproduction instructions, and @fvictorio for figuring out what was going on.
